bagisto/src/main/java/Abstract/AbstractComponen.java

```diff
@@ -67,6 +75,16 @@ public void mouseHover(WebElement findBy) {
 		actionClass.moveToElement(findBy).build().perform();
 	}
 	
+	public Select selectDropdown(WebElement selectdropDown) {
+		Select selectDrop = new Select(selectdropDown);
+		return selectDrop;
+	}
+	
+	public Float remove$(String price) {
+	   Float a = Float.parseFloat(price.replaceAll("[-+$^]*", ""));
```

Comment:
Remo comma from the given string,
If the price will Like $10,250  then it will give error

Reply:
resolved